<launch>
    <node pkg="pedstrian_detection_opencv" name="pedstrian_detection_opencv_node" type="pedstrian_detection_opencv_node" output="screen" respawn="true">
        <param name="subscribed_topic" type="string" value="/hk_video"/>
        <param name="show_video_flag" type="bool" value="true"/>
        <param name="save_result_video_flag" type="bool" value="false"/>
        <param name="result_video_file_name" type="string" value="/home/zj/ros_wl/src/pedestrian_detection_tracking/pedstrian_detection_opencv/video_output/raw.avi"/>
        <param name="rate" type="double" value="10.0"/>
    </node>
</launch>
